### PR TITLE
feat: ACME Challenge completer worker

### LINKED ISSUE #170: create queue worker for ACME challenge completion
Use appropriate method of let's encrypt class

This fills out step 3 of
https://github.com/Seneca-CDOT/starchart/issues/167

### PR DESCRIPTION
**Pull request by dadolhay**
The implemented worke:
- runs after we verify that the DNS challenge record has been propagated in Route53
- completes all challenges with the ACME provider

Closes #170

### CODE REVIEW

File: app/queues/certificate/challenge-completer-worker.server.ts [hunk 1]
```diff
@@ -13,7 +19,84 @@ export const challengeCompleterWorker = new Worker<ChallengeCompleterData>(
...
+
+    // Get the order creator return value
+    const [orderCreatorRetval] = Object.values(childrenValues);
+
+    logger.info('Attempting to complete ACME challenges with the provider', {
+      rootDomain,
+      certificateId: orderCreatorRetval.certificateId,
+    });
+
+    let letsEncrypt;
```
Comment (humphd) [inline app/queues/certificate/challenge-completer-worker.server.ts:35]: Can you move this lower down, where you actually use it near 60?

**Review by humphd — DISMISSED** [review]
Looks good, one nit.

File: app/queues/certificate/challenge-completer-worker.server.ts (hunk 1 above)
Comment (dadolhay) [reply app/queues/certificate/challenge-completer-worker.server.ts]: Sure!

**Review by humphd — APPROVED** [review]

**Review by Genne23v — APPROVED** [review]
Looks good to me. I think I should add some log to DNS module too.

**Pull request merged**